neuralmonkey/encoders/sentence_encoder.py

```diff
@@ -117,6 +117,10 @@ def _attention_mask(self):
         return self.input_mask
 
     @property
+    def states_mask(self):
+        return self.input_mask
+
```

Comment:
Já bych to tyhle rozdíly teď neřešil a někdy se sejdeme a rozmyslíme pořádnou hierarchii a dědičnost těch částí modelu.